test/browser/integration/caniuse.js

```diff
@@ -65,6 +65,7 @@ window.caniusecb = function(caniuse) {
       geolocation: 'geolocation',
       getrandomvalues: 'getrandomvalues',
       getusermedia: 'stream',
+      hairline: 'hairline',
```

Comment:
I am not [seeing this on caniuse](http://caniuse.com/#search=hairline) - can you remove unlessI am missing something